src/ros_can_nodes/CMakeLists.txt

```diff
@@ -7,12 +7,14 @@ add_compile_options(-std=c++17 -O2)
 ## Find catkin macros and libraries
 ## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
 ## is used, also find other catkin packages
-find_package(catkin REQUIRED COMPONENTS
-  ros_type_introspection
+set(MESSAGE_PACKAGES
   std_msgs
-  topic_tools
   owr_messages
 )
```

Comment:
is there a way we could automate the generation of this list (maybe using python?)

Reply:
do you mean to just find all possible packages with messages?

Reply:
Yeah. This is something we don't want to have to recompile all the time

Reply:
it doesnt seem to recompile on consecutive runs of `catkin_make`. the only times that this map needs to be recompiled is when a new message package is added or a message is changed

Reply:
Yeah that's what I mean - we don't want to have to keep adding packages here unless we add new packages to the system. Ideally people working on other systems shouldn't have to know how to edit this